.jekyll-cache/Jekyll/Cache/Jekyll--Converters--Markdown/6b/23ae3ebb8167a6890eb99ca3aea578e01099d30bd0450b33d75bbc5703eb43

```diff
@@ -0,0 +1,36 @@
+I"�<h1 id="ict-baseline-alignment-framework---test-cases">ICT Baseline Alignment Framework - Test Cases</h1>
+<h2 id="placeholder-index">Placeholder Index</h2>
+
+<p>Until we modify the primary index file, this serves as a placeholder with links to current test case documents.</p>
+
+<p>Test Case documents are organized by Baseline Test (each serving as a Test Scenario or general “functional requirement” for testing), which inculde multiple Test Cases. Will need to develop a Test Scenario template to summarize Test Scenario purpose and subordinate Test Cases</p>
+
+<ul>
+  <li><code class="language-plaintext highlighter-rouge">5.</code> Changing Content
+    <ul>
+      <li><a href="testcases/TC05.1-1-fail-1.html">TC05.1-1-fail-1</a></li>
+    </ul>
+  </li>
+  <li><code class="language-plaintext highlighter-rouge">6.</code> Images
+    <ul>
+      <li><a href="testcases/TC06-001-fail.html">TC06-001-fail</a></li>
+      <li><a href="testcases/TC06.1-1-fail-1.html">TC06.1-1-fail-1</a> (new version of file/Test Case above, but with revised Test Case numbering)</li>
```

Comment:
I didn't include a note about the revised test case numbering for 5.1.  I don't think it's needed. But, if it is, we should be consistent...

Reply:
We still need to go back and revise the other test cases for consistency, so we need to add an issue/task for that. Definitely should be consistent, and I think we'll want to make sure we address in our "Contributing" guidance.

Reply:
With the limited number of test cases that need an ID update, I don't think the note is needed. 

Yes, will need to add how to name a TC and TF in guidance. Maybe, for each issue to create a TC and TF, the filenames can be listed.